addons/options/README.md

```diff
@@ -44,6 +44,7 @@ setOptions({
   showSearchBox: false,
   downPanelInRight: false,
   sortStoriesByKind: false,
+  hierarchySeparator: /\//,
```

Comment:
ah cool, this would get applied automatically - what are the chances this PR will make it into 3.2? if the chances are low I will adapt the readme and the definitelytyped PR.